api/python/provisioner/inputs.py

```diff
@@ -1113,23 +1113,26 @@ def pillar_value(self):
             return res
         elif self.is_remote():
             return {
-                f'os': {
-                    'source': (f'{self.source}/os'),
+                f'{config.OS_ISO_DIR}': {
```

Comment:
just `config.OS_ISO_DIR: ...` should be good

Reply:
Fixed